lib/matplotlib/quiver.py

```diff
@@ -85,6 +85,13 @@
     height of the axes, respectively, when the the window is resized;
     for 'dots' or 'inches', resizing does not change the arrows.
 
+  *orientation*: ['absolute' | 'relative']
```

Comment:
Nice clear docstring. :+1: 
Out of interest though, what is the purpose of the absolute mode (other than that being the current behaviour)? When would one want to specify coordinates in data space, but vectors in non-data space (since the x and y axes may be inverted)?